src/containers/org/members/utils/memberHasSelectedIdentityTypes.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import { List, Map } from 'immutable';
+import type { Role } from 'lattice';
+
+import {
+  AUTH0,
+  SAML,
+  SAMLP,
+  SOCIAL
+} from './constants';
+
+export default function memberHasSelectedIdentityTypes(member :Map, selectedIdentityTypes :Role[]) {
```

Comment:
seems fine on my end?

Reply:
I mean that I don't think `selectedIdentityTypes` is actually an array of roles. It's an array, but it seems like an array of strings because ...

```
identityType === SOCIAL
identityType === AUTH0
identityType === SAML
```